chatkit-core/src/main/kotlin/com/pusher/chatkit/CurrentUser.kt

```diff
@@ -89,6 +159,15 @@ class CurrentUser(
             callback: (Result<Room, Error>) -> Unit
     ) = makeCallback({ syncCurrentUser.createRoom(id, name, pushNotificationTitleOverride, isPrivate, customData, userIds) }, callback)
 
+    /**
+     * Update an existing [room]'s properties. If an argument is omitted or provided as null,
+     * the existing value will not be changed.
+     * - [name] - a String to label the room
```

Comment:
Oh god, this is supposed to be optional! (not a problem with the docs, but a mistake here in `CurrentUser`. It's correctly optional in `SynchronousCurrentUser#updateRoom`)

Reply:
Oh! the name is supposed to be optional? I'll add to my technical debt backlog.